Website/Packages/datax-pipeline/src/modules/flowDefinition/flowActions.js

```diff
@@ -102,6 +110,12 @@ export const initFlow = context => (dispatch, getState) => {
     } else {
         const owner = UserSelectors.getUserAlias(getState());
         dispatch(QueryActions.initQuery(QueryModels.defaultQuery));
+        isDatabricksSparkType().then(response => {
+            return dispatch({
```

Comment:
Once you created updateIsDatabricksSparkType, you can use it here as well.

Reply:
done